ironic/drivers/modules/drac/management.py

```diff
@@ -357,3 +358,32 @@ def get_sensors_data(self, task):
                   sensor type, which can be processed by Ceilometer.
         """
         raise NotImplementedError()
+
+    @METRICS.timer('DracManagement.reset_idrac')
+    @base.clean_step(priority=CONF.drac.clean_priority_reset_idrac)
+    def reset_idrac(self, task):
+        """Reset the iDRAC configuration.
+
+        :param task: a TaskManager instance containing the node to act on.
+        :returns: None if it is completed.
+        :raises: DracOperationError on an error from python-dracclient.
+        """
+        node = task.node
+
+        client = drac_common.get_drac_client(node)
+        client.reset_idrac(force=True, wait=True)
```

Comment:
I believe you need to tell ironic conductor to resume cleaning after the reset completes like this:
manager_utils.notify_conductor_resume_clean(task)